src/libecalc/core/graph_result.py

```diff
@@ -45,6 +46,40 @@ class EnergyCalculatorResult(BaseModel):
     variables_map: dto.VariablesMap
 
 
+# TODO: Move to "DTO"..this is presentation layer stuff
+
+
+def convert_to_timeseries(
+    emission_core_results: Dict[str, Dict[str, EmissionResult]],
+    regularities: Union[List[float], Dict[str, List[float]]],
+) -> Dict[str, Dict[str, PartialEmissionResult]]:
+    """
+    Emissions by installation and name
+    Args:
+        emission_core_results:
+        regularities:
+
+    Returns:
+
+    """
+    dto_result: Dict[str, Dict[str, PartialEmissionResult]] = {}
+
+    for installation_name, emissions in emission_core_results.items():
```

Comment:
emission_core_results are per consumer, not per installation?

Reply:
I just happened to got error on that in the test, so yes, it seems like you are right :)

Reply:
fixed